t/stream-plugin/mqtt-proxy.t

```diff
@@ -220,7 +220,56 @@ hello world
 
 
 
-=== TEST 9: set route with invalid host
+=== TEST 9: set route with IP CIDR
```

Comment:
New test should be added from the bottom of the file. And it should be in stream-node/sanity.t